### DISCUSSION TITLE
feature: feedback

### DISCUSSION THREAD

**Discussion by h0rban** [sandboxnu/office-hours#571] [Ideas]
It would be nice to have a feature that allows students to provide tas with feedback after their call.

**Comment by willstenzel** [comment]
@h0rban thanks for your feature suggestion! That's a good idea and we've actually considered adding that feature. However, we had concerns that some TA's might not enjoy having constant feedback through the app so we figured it would be best to leave it up to the course coordinators to collect feedback on the TA's in the way they found best.

**Comment by willstenzel** [comment]
Out of curiosity, @h0rban do you happen to be a TA? I'm curious to hear more thoughts on what TA's think about this.

**Reply by h0rban** [reply]
Yes, I am a TA and have used the app as a student before. I am not sure that Instructors are currently collecting data specifically on TA's performance, and I agree that it should be up to them. An idea I had in mind was similar to rating your ride after uber and summary statistics might only be visible to the instructor. There could a survey a student takes after receiving help and rates several questions on the Likert scale. Questions could include the student's level of preparation, how many of them a ta could help with etc. Additionally, there could be an option to notify an instructor if a student felt offended during office hours, for example
